gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -496,25 +543,39 @@ public CallOptions getCallOptions() {
     return callOptions;
   }
 
+  /** Backport of {@link #getStreamWaitTimeoutDuration()} */
+  @Override
+  @Nullable
+  @ObsoleteApi("Use getStreamWaitTimeoutDuration() instead")
+  public org.threeten.bp.Duration getStreamWaitTimeout() {
+    return toThreetenDuration(getStreamWaitTimeoutDuration());
+  }
+
   /**
    * The stream wait timeout set for this context.
    *
-   * @see ApiCallContext#withStreamWaitTimeout(Duration)
+   * @see ApiCallContext#withStreamWaitTimeoutDuration(java.time.Duration)
    */
   @Override
   @Nullable
-  public Duration getStreamWaitTimeout() {
+  public java.time.Duration getStreamWaitTimeoutDuration() {
     return streamWaitTimeout;
   }
 
+  @Override
+  @Nullable
+  @ObsoleteApi("Use getStreamIdleTimeoutDuration() instead")
```

Comment:
same comment as above - add this to the Javadoc as well

Reply:
Done. I modified all methods marked with `@ObsoleteApi` to include a similar javadoc